### ISSUE TITLE
Autofill issue with Firefox 83.1 (Android 9/10)

### ISSUE THREAD

**Issue by daviddem1971** [bitwarden/mobile#1167]
## Describe the Bug

It seems enabling accessibility in BW interferes with autofill in Firefox.
## Steps To Reproduce

1. in BW autofill services, enable autofill, disable accessibility
2. open Google sign-in in Firefox (or any other site with matching logins in your vault)
3. as expected, obtain a BW pop up with possible matches from the vault
4. in BW autofill, enable autofill, enable accessibility, enable draw over
5. open Google sign-in in Firefox
6. now the possible matches from the vault do not show up. Only the "go to my vault" option:
## Expected Result

Expected that Firefox would favor autofill over accessibility, as do other browsers and apps
## Actual Result

Autofill does not work in Firefox when accessibility is enabled in BW
## Screenshots or Videos

Firefox with autofill enabled and accessibility disabled in BW:

![Screenshot 1](https://user-images.githubusercontent.com/47311661/101362327-421c3600-38a0-11eb-8c49-0994b0dc26d8.png)

Firefox with autofill, accessibility and draw over enabled in BW:

![Screenshot 2](https://user-images.githubusercontent.com/47311661/101362463-6c6df380-38a0-11eb-8e12-81fb871ac11e.png)
## Environment

- Device: Samsung S9+ / Samsung A20e
- Operating system: Android 9 / Android 10
- Firefox 83.1 or Firefox 84 beta
- BW app 2.7.0 (3258)

## Additional Context

I did try the above with the BW extension for Firefox enabled or disabled, still same result.
